solenoid/robot.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+import wpilib
+from wpilib.shuffleboard import Shuffleboard
+
+"""This is a sample program showing the use of the solenoid classes during operator control. Three
+    buttons from a joystick will be used to control two solenoids: One button to control the position
+    of a single solenoid and the other two buttons to control a double solenoid. Single solenoids can
+    either be on or off, such that the air diverted through them goes through either one channel or
+    the other. Double solenoids have three states: Off, Forward, and Reverse. Forward and Reverse
+    divert the air through the two channels and correspond to the on and off of a single solenoid,
+    but a double solenoid can also be "off", where the solenoid will remain in its default power off
+    state. Additionally, double solenoids take up two channels on your PCM whereas single solenoids
+    only take a single channel.
+"""
+
+
+class MyRobot(wpilib.TimedRobot):
+    def robotInit(self):
+        self.joystick = wpilib.Joystick(0)
+
+        # Solenoid corresponds to a single solenoid.
+        # In this case, it's connected to channel 0 of a PH with the default CAN ID.
+
```

Comment:
Generally, I like my comments describing a specific statement attached to the statement:

```python
        # Solenoid corresponds to a single solenoid.
        # In this case, it's connected to channel 0 of a PH with the default CAN ID.
        self.solenoid = wpilib.Solenoid(
            moduleType=wpilib.PneumaticsModuleType.REVPH, channel=0
        )
```

But with a comment that is describing a group of statements I'll detach it.

```python
        # Publish elements to shuffleboard

        tab = Shuffleboard.getTab("Pneumatics")
        tab.add("Single Solenoid", self.solenoid)
        tab.add("Double Solenoid", self.doubleSolenoid)
        tab.add("Compressor", self.compressor)
````

Reply:
Okay. Those are great suggestions. I will do my best to implement and then commit again.

Reply:
I already did it. :)